plugin/trino-kafka/pom.xml

```diff
@@ -141,6 +135,13 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
We should be able to remove this, since these annotations are only useful for checkers, not actually needed when running.

Reply:
I've removed only "@Nullable" usage in test code. It isn't needed and doesn't add any additional information there.